### DISCUSSION TITLE
Markdown Header Links Possibly Broken in Preview?

### DISCUSSION THREAD

**Discussion by NuclearPowerNerd** [dendronhq/dendron#3529] [Q&A]
Hi,

I am using dendron version `v0.112.0` with VSCode version `1.71.1`

I use links within markdown files to help navigate large markdown files. For example the top of my markdown file might look like

```
## Table of Contents

* [Topic 1](#topic-1)
* [Topic 2](#topic-2)
* [Topic 3](#topic-3)
* [Topic 4](#topic-4)
* [Topic 5](#topic-5)
* ...and so on
```

I can then use these links within the preview to jump around to different parts of the file which is very convenient for large files. This worked fine until the update to the version stated above. Now my links no longer work in the preview.

Any ideas?

**Comment by jonathanyeung** [comment]
Hi @NuclearPowerNerd - thanks for reporting the issue. A few questions:

- What does the header look like? Is it `# topic-1`?
- Are you using the Dendron preview or VS Code's built-in Markdown preview?
- Did this stop working after you upgraded Dendron or after you upgraded VSCode?

**Reply by NuclearPowerNerd** [reply]
Thanks for the quick reply.

It looks like the following (an abbreviated example). This problem occurs for any level of header.

```
# Table of Contents

* [Topic 1](#topic-1)


## Topic 1

Blah blah blah
```
I am using the Dendron Preview in VSCode by doing `Dendron: Toggle Preview`

I _think_ as early as Monday of last week (Sept 5th 2022) it was working fine. I am sorry I can't be more specific than that but I don't remember. 

Just to note, I have been using Dendron for over a year and in all that time it has always worked (clicking a link within a page and then the view scrolls down to that section). 

Please let me know if there is anything else I can provide.

**Reply by jonathanyeung** [reply]
I can repro the issue - looks like this regression was introduced in `v0.112.0`.  Let me investigate a fix - in the meantime, you can downgrade to v 111 to get back the functionality if you need. 

Thanks for reporting the issue!

**Reply by jonathanyeung** [reply]
@NuclearPowerNerd - this should be fixed in the upcoming release v `0.113.0`, which should come out tomorrow.
